usagov/usagov-2021
# Usagov 1866 elected officials textarea size

## Jira Task
Elected officials textarea size


https://cm-jira.usa.gov/browse/USAGOV-1866

## Description

The goal of this change is to set the size of the textareas on the Email Elected Officials page.
I also took the opportunity to convert a couple of elected officials page into custom twig templates.

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions


- First edit these 4 pages by deleting everything from the body field and checking the box in the Custom Twig Content field:
  - /elected-officials-email
  - /elected-officials-results
  - /es/funcionarios-electos-correo-electronico
  - /es/funcionarios-electos-resultados
- Clear all caches
- View the pages to make sure everything looks OK
- Confirm that the textareas on these 2 pages are full width (should be 320px) and tall enough for 5 rows of text.
  - /elected-officials-email
  - /es/funcionarios-electos-correo-electronico
- You can also add these parameters at the end of any of those urls to view the pages with results. (But this PR should not effect those parts of the pages)
  - `?input-street=1800+F+ST+NW&input-city=WASHINGTON&input-state=DC&input-zip=20270&email=pmendelson%40dccouncil.gov&name=Phil+Mendelson&office=D.C.+City+Council+Chairman#skip-to-h1`

### Change Requirements


- [x] Requires Updated Content

### Validation Steps

- Test instruction 1
- Test instruction 2
- Test instruction 3

## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
